app/(afterLogin)/(employeeInformation)/employees/manage-employees/page.tsx

```
@@ -67,23 +71,29 @@ const ManageEmployees: React.FC<any> = () => {
   const getFilterDisplayName = (key: string, value: string) => {
     switch (key) {
       case 'allOffices':
-        const office = EmployeeBranches?.items?.find((item: any) => item.id === value);
+        const office = EmployeeBranches?.items?.find(
+          (item: any) => item.id === value,
+        );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)